README.md

```diff
@@ -20,6 +20,23 @@ CephFS CSI plugins, see documentation in `docs/`.
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
```

Comment:
@poornimag ideally we should have these columns :  
Branch/CSI driver Version, CSI Spec , variants like RBD and CephFS, Kubernetes version